java/main/src/main/java/com/saucelabs/saucebindings/SauceSession.java

```diff
@@ -334,8 +385,15 @@ private void validateSessionStarted(String method) {
   }
 
   private void quit() {
-    driver.quit();
-    driver = null;
+    try {
+      driver.quit();
+    } catch (WebDriverException e) {
```

Comment:
This probably can be more than just WebDriverException. We really want the session to stop if we use this method, and this is a backup to sending the quit command. We do want this because if the driver is working, we want the quit command in the logs with timestamp, which you don't get if you just call stop via API.

Reply:
I changed this to Exception